client/core/status.go

```diff
@@ -21,7 +22,7 @@ func statusResolutionID(dc *dexConnection, trade *trackedTrade, match *matchTrac
 // conflicts. resolveMatchConflicts will block until the call to match_status
 // returns, but then goroutines are started to handle the actual resolution.
 // The trackedTrades' mutexes will remain locked until resolution completes.
```

Comment:
I'm not sure the docs here are still accurate since it waits for the resolution goroutines now, making this detail unimportant to the caller.